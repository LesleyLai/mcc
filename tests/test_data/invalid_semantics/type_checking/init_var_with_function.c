int f(void);

int main(void)
{
  int x = f;
}
