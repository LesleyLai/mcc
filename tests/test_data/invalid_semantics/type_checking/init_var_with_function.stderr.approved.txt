{{filename}}:4:11: Error: initialization of 'int' from 'int(void)'
4 |   int x = f;
  |           ^

